src/pages/layer-2.js

```diff
@@ -0,0 +1,614 @@
+// Libraries
+import React from "react"
+import { graphql } from "gatsby"
+import { getImage } from "gatsby-plugin-image"
+import styled from "styled-components"
+import { useIntl } from "gatsby-plugin-intl"
+
+// Assets
+import privacyGlyph from "../assets/run-a-node/privacy-glyph.svg"
+
+// Data
+import cexSupport from "../data/layer-2/cex-layer-2-support.json"
+import layer2Data from "../data/layer-2/layer-2.json"
+import validiumData from "../data/validium.json"
+
+// Components
+import ExpandableCard from "../components/ExpandableCard"
+import InfoBanner from "../components/InfoBanner"
+import Link from "../components/Link"
+import PageHero from "../components/PageHero"
+import PageMetadata from "../components/PageMetadata"
+import Pill from "../components/Pill"
+import ProductCard from "../components/ProductCard"
+import { CardGrid, Content, Page } from "../components/SharedStyledComponents"
+
+// Utils
+import { translateMessageId } from "../utils/translations"
+
+// Styles
+const GappedPage = styled(Page)`
+  gap: 4rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    gap: 3rem;
+  }
+  * {
+    scroll-margin-top: 5.5rem;
+  }
+`
+
+const HeroContainer = styled.div`
+  background: ${({ theme }) => theme.colors.runNodeGradient};
+  width: 100%;
+`
+
+const Hero = styled(PageHero)`
+  padding-bottom: 2rem;
+`
+
+const Flex50 = styled.div`
+  flex: 50%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex: 100%;
+  }
+`
+
+const TwoColumnContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  gap: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 0rem;
+    margin-right: 0rem;
+  }
+`
+
+const InfoGrid = styled(CardGrid)`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(min(100%, 340px), 1fr));
+  gap: 1rem 2rem;
+  & > div {
+    height: fit-content;
+    &:hover {
+      transition: 0.1s;
+      transform: scale(1.01);
+      img {
+        transition: 0.1s;
+        transform: scale(1.1);
+      }
+    }
+  }
+`
+
+const StyledCardGrid = styled(CardGrid)`
+  margin-bottom: 4rem;
+  margin-top: 4rem;
+`
+
+const Layer2Page = ({ data }) => {
+  const intl = useIntl()
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
+    subtitle: "Smart solutions to scale Ethereum",
+    image: getImage(data.ethBlocks),
+    alt: "test",
+    buttons: [
```

Comment:
For these buttons...

![Image 2022-03-12 at 3 35 04 PM](https://user-images.githubusercontent.com/8097623/158038681-094d91f9-9758-470a-8018-467b1fa5f0d6.jpg)

I like the idea of:
- "Use a layer 2" --> scrolls straight to section on how to use a layer 2 & comparing options
- "What is layer 2" --> scrolls below fold to section that explains what they are